pkg/apis/app/v1alpha1/application_types.go

```diff
@@ -78,6 +82,23 @@ type ApplicationStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 }
 
+// ImageSpec contains information about an image used as an icon.
+type ImageSpec struct {
+
```

Comment:
this newline doesn't exist on any other type. yay for nitpicking

Reply:
Removed extra line. Thanks for catching the consistency issue.